.cirrus.yml

```diff
@@ -430,8 +431,10 @@ task:
   setup_script:
     - date
     - which flutter
-    - sudo gem install cocoapods
-    - sudo gem install xcpretty
+    - ruby --version
+    - sudo gem install bundler -v 2.0.2 -N
+    - bundle install --system --gemfile=dev/ci/mac/Gemfile
+    - cat dev/ci/mac/Gemfile.lock
```

Comment:
What does this do? Should it be `cat dev/ci/mac/Gemfile.lock > <file_path>`? I can see that we're printing it here but not that it's being saved or used anywhere.

Reply:
This is debugging that I already removed in a commit I have locally, but didn't want to push to re-trigger CI. Before landing I will remove this, it was only so I could see it in cirrus.